README.md

```diff
@@ -18,7 +18,38 @@ If you don't have an app already, you can create a new web API project using the
 dotnet new web
 ```
 
-### Step 2: Add NuGet Package Reference
+### Step 2: Generate a PAT
```

Comment:
Just a suggestion, use numbered list instead. That you don't need to worry about keep the number correct.

For example, `1. Generate a PAT / 1. Add NuGet Source` will be rendered in proper numbers as:

```
1. Generate a PAT
2. Add NuGet Source
```